### PR TITLE
Dala 49 initial edc

### PR DESCRIPTION
**Pull request by arjmandi-hamid**
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [ ] The code has been manually inspected
- [ ] The new feature was observed "in running software"
- [ ] The DoD was checked for completeness
- [ ] The PR actually implements what is described in the JIRA-Issue
- [ ] At least one E2E Test exists testing the new feature
- [ ] Documentation is updated as required
- [ ] The new version is deployed to preview using this branch
  - [ ] It's verified that this version actually is the one deployed (check actuator/info for branch name and commit id!)
  - [ ] The new feature is manually used/tested/observed on preview
- [ ] All Classes and Interfaces contain JavaDoc

### CODE REVIEW

File: dataland-edc-dummyserver/src/test/kotlin/org/dataland/edcDummyServer/DummyEdcControllerTest.kt [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            MockMvcRequestBuilders.get("/dataland/data/1")
+        ).andExpectAll(
+            MockMvcResultMatchers.status().isOk,
+            MockMvcResultMatchers.content().contentType(MediaType.APPLICATION_JSON),
+            MockMvcResultMatchers.content().string(body)
+        )
+    }
+
+    @Test
+    fun `abcd`() {
```
Comment (SiegfriedSobkowiak) [inline dataland-edc-dummyserver/src/test/kotlin/org/dataland/edcDummyServer/DummyEdcControllerTest.kt:52]: Die Tests hier am Ende sollten raus da sie nichts sinnvolles tun.

File: .github/workflows/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+- [ ] The code has been manually inspected
+- [ ] The new feature was observed "in running software"
+- [ ] The DoD was checked for completeness
+- [ ] The PR actually implements what is described in the JIRA-Issue
+- [ ] At least one E2E Test exists testing the new feature
+- [ ] Documentation is updated as required
+- [ ] The new version is deployed to preview using this branch
+  - [ ] It's verified that this version actually is the one deployed (check actuator/info for branch name and commit id!)
+  - [ ] The new feature is manually used/tested/observed on preview
+- [ ] All Classes and Interfaces contain JavaDoc
```
[the remark this answers is not in the document]
Comment (emanueldfine) [reply .github/workflows/pull_request_template.md]: Letzter Punkt entfernt

File: dataland-edc-dummyserver/src/test/kotlin/org/dataland/edcDummyServer/DummyEdcControllerTest.kt (hunk 1 above)
Comment (arjmandi-hamid) [reply dataland-edc-dummyserver/src/test/kotlin/org/dataland/edcDummyServer/DummyEdcControllerTest.kt:52]: Zwei Pinkte: 
1. Wenn es in Sonar gibt, bleibt es noch grün.
2. Sonar beschwert sich über Namen der test funktionen wenn es leerzeichnen hat. da habe ich nur das getestet. Weisst du wiesio es so ist?

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+version: "3.8"
+services:
```
[the remark this answers is not in the document]
Comment (SiegfriedSobkowiak) [reply docker-compose.yml]: wurde entfernt

File: config/baseline.xml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" ?>
+<SmellBaseline>
+  <ManuallySuppressedIssues></ManuallySuppressedIssues>
+  <CurrentIssues>
+    <ID>SpreadOperator:DatalandBackend.kt$(*args)</ID>
```
[the remark this answers is not in the document]
Comment (SiegfriedSobkowiak) [reply config/baseline.xml]: entfernt

File: .github/workflows/DummyEDC.yaml [hunk 5]
```diff
@@ -0,0 +1,60 @@
+name: DummyEDC
```
[the remark this answers is not in the document]
Comment (SiegfriedSobkowiak) [reply .github/workflows/DummyEDC.yaml]: geändert

File: .github/workflows/CI.yaml [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+        with:
+          name: JaCoCoEDCDummyUnitTests
+#      - uses: actions/download-artifact@v2
+#        with:
+#          name: JaCoCoAPIUnitTests
+      - name: Generate jacocoTestReport and run Sonar
+        if: always()
+        run: |
+          ./gradlew jacocoTestReport --no-daemon
+          find . -type f -iname "*.exec";
```
[the remark this answers is not in the document]
Comment (SiegfriedSobkowiak) [reply .github/workflows/CI.yaml]: geändert

File: dataland-edc-dummyserver/src/test/kotlin/org/dataland/edcDummyServer/DummyEdcControllerTest.kt (hunk 1 above)
Comment (SiegfriedSobkowiak) [reply dataland-edc-dummyserver/src/test/kotlin/org/dataland/edcDummyServer/DummyEdcControllerTest.kt]: Tests entfernt, code smells blocken Sonar nicht

**Pull request merged**